### ISSUE TITLE
Digital Sensor Node (Button) not working

### ISSUE THREAD

**Issue by pleemans67** [O-Hahn/node-red-contrib-grovepi#3]
The Grove Digital Sensor (Button)  & Grove digital event does not seem to work. Any Tips what could be wrong?

**Comment by O-Hahn** [comment]
have you installed the latest version - i have provided a fix at version 0.1.8 ..

**Comment by pleemans67** [comment]
Yes. In the mean time i did some further investigation. I think the issue is related to RPI 3B model(Timing issue with the GrovePi). I will do a test with an RPI model2.

**Comment by O-Hahn** [comment]
yes i have only tested on RPI 3 and RPI 2. I do not have a 3B up to now...

**Comment by pleemans67** [comment]
I did some further analysis, here are my findings.

Using an RPI3 & grovepi with firmware 1.2.7:
- A simple python script reading digital input does not work.
- The Node Red Digital Sensor (Button) does not work.
- the Node Red Grove digital event does not work.

Using an RPI 3 & grovepi **with latest firmware 1.3.0**:
- A simple python script reading digital input  **works**.
- The Node Red Digital Sensor (Button) does not work.
- the Node Red Grove digital event does not work.

Using an RPI 2 & grovepi with firmware 1.2.7:
- A simple python script reading digital input  **works**.
- The Node Red Digital Sensor (Button)  **works**.
- the Node Red Grove digital event does not work.

My conclusion : seems to be timing dependent?

Note: The Grove digital event node never works, can you confirm this?

Note2: you replied that it works on RPI3, but i have other findings. Any tips?
